go.mod

```diff
@@ -55,7 +55,6 @@ require (
 	k8s.io/api v0.20.0
 	k8s.io/apiextensions-apiserver v0.20.0
 	k8s.io/apimachinery v0.20.0
-	k8s.io/apiserver v0.20.0
```

Comment:
I was looking through the codebase and noticed that the apiserver dependency was unused. It looks like running `go mod tidy` locally backed up this assumption.

Reply:
We used to vendor more k8s stuff in this repo and had a bit of circular deps between OLM and registry. We decoupled them and resolved those issues. So this is probably leftover that is not cleaned-up.

Reply:
Yeah, that's understandable. I was looking through the codebase and noticed that, and I have Go 1.16 locally which was able to spot that in the go.mod. This could also just be an artifact from the Go 1.15 bump from last week.

Reply:
Actually, this kind of scenario is exactly why I added a `vendor` check in the metering-related repositories. I can add something similar as an action or update the prow configuration.

Reply:
Nice, it looks like #596 is already producing CI feedback that prompted this PR. It looks like there may be some race conditions in the bundle unpacking test that are popping up this week. We need to merge this change first before attempting to merge that vendor action check.